### PR TITLE
Add JoinGSCButton to PortfolioCard on the OverviewPage

### PR DESCRIPTION
**Pull request by sentilesdal**
image

### CODE REVIEW

File: src/ui/gsc/JoinGSCButton.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  const votePower = useVotingPowerForAccountAtLatestBlock(account);
+  const { data: threshold = BigNumber.from(0) } = useGSCVotePowerThreshold();
+  const isOnGSC = useIsGSCMember(account);
+
+  const hasEnoughToJoinGSC = parseEther(votePower).gte(threshold);
+  const canLeaveGSC = isOnGSC && parseEther(votePower).lt(threshold);
+
+  const handleJoin = useHandleJoin(account, signer);
+  const handleLeave = useHandleLeave(account, signer);
```
Comment (DannyDelott) [inline src/ui/gsc/JoinGSCButton.tsx:45]: minor naming nit: `useJoinGSC` and `useLeaveGSC`

File: src/ui/gsc/useIsGSCMember.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    "members",
+    {
+      callArgs: [account as string],
+      enabled: !!account,
+    },
+  );
+
+  // if there is a valid timestamp, the user is on the GSC, if it is zero, that indicates there is
+  // no entry.
+  return !!timeStamp?.toNumber();
```
Comment (DannyDelott) [inline src/ui/gsc/useIsGSCMember.tsx:16]: let's use `select` here as well

**Review by DannyDelott — APPROVED** [review]

**Comment by sentilesdal** [comment]
#### 📝 Changed routes:


| [**/gsc-proposals**](https://elf-council-frontend-staging-nmc6uxj67-element-finance.vercel.app/gsc-proposals) | [**/proposals**](https://elf-council-frontend-staging-nmc6uxj67-element-finance.vercel.app/proposals) |
|:-:|:-:|
|<a href="https://elf-council-frontend-staging-nmc6uxj67-element-finance.vercel.app/gsc-proposals">Screenshot of /gsc-proposals</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/AeD8I_4bVnUCUwCumuNPlNuMBHDPi_uawh4Xd5uvYrgz1v6rGPcKhUgVQvaVARSgeMsOfKCXJX2R2diKoINBNQ.png">(view full size)</a> |<a href="https://elf-council-frontend-staging-nmc6uxj67-element-finance.vercel.app/proposals">Screenshot of /proposals</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/Kw8Cq_9YF0RQTYoA6oD-VbchgClb_u2kjvWmtYy5x3kcp6ucVmT0XdNxRy9OeiW8xVwrK-fjNwd9-_n2XEzjpA.png">(view full size)</a> |







Commit d5930a66ca338e86befc992967e05588a7374d7b (https://elf-council-frontend-staging-nmc6uxj67-element-finance.vercel.app).

File: src/ui/gsc/useIsGSCMember.tsx (hunk 2 above)
Comment (sentilesdal) [reply src/ui/gsc/useIsGSCMember.tsx:16]: yes! here makes sense

**Pull request merged**